website-html-to-markdown/moveover/testing/00-testing-new-designs/typography.md

```diff
@@ -0,0 +1,64 @@
+<section class="doc-page-temp-testing-typography">
+    <kbd>&lt;h1 class="doc-text-h1"&gt;</kbd>
```

Comment:
Super Nit: The `code` element seems more appropriate here. (`kbd` is meant to represent keyboard input.)

(I know this is just sample code to show the visual designs though.)

Reply:
yes, the reason why I used `kbd` instead of `code` is because `<code>` is styled in markdown and I didn't want to mix & match tags, so I picked one HTML tag that was "easy" ;) 

in any case it will be deleted later, is only for testing